### DISCUSSION TITLE
language settings

### DISCUSSION THREAD

**Discussion by quelsemme** [logancyang/obsidian-copilot#215] [Q&A]
I don't use this function often, but when I tried to convert a note into a tweet or a tweet thread, the tweets come out in Spanish? My notes are in English (and my non-English notes are in Italian). Any idea what is going on?

The rest of Copilot appears to be English.

**Comment by logancyang** [comment]
@quelsemme Thanks for reporting! Were you using gpt-3.5? It used to do this when I tried as well. Some models need more specific prompting to behave as expected. You can make your own custom prompts to experiment. I plan to retire some of these builtin prompts soon.

**Reply by quelsemme** [reply]
I was Using GPT-4 TURBO.
GPT-4worked in english.
